UnityProject/Assets/Scripts/Core/SecureStuff/TTSCommunication.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Logs;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace SecureStuff
+{
+	public static class TTSCommunication
+	{
+		private class DataTSS
+		{
+			public string input_string { get; set; }
+			public string voice { get; set; }
+		}
+
+		public static async Task<byte[]> GenTTS(string Input, string voice)
+		{
+			// URL of the API endpoint
+			string url = "http://127.0.0.1:5234/generate-audio";
```

Comment:
shouldn't this be pre-configured in the game config?

Reply:
will always be using local server and this is secure stuff so shouldn't be able to be modified